docs/en/engines/table-engines/integrations/deltalake.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: /en/engines/table-engines/integrations/deltalake
+sidebar_label: DeltaLake
+---
+
+# DeltaLake Table Engine
+
+This engine provides a read-only integration with existing [Delta Lake](https://github.com/delta-io/delta) tables in Amazon S3.
+
+## Create Table
+
+Note that the Delta Lake table must already exist in S3, this command does not take DDL parameters to create a new table.
+
+``` sql
+CREATE TABLE deltalake
+    ENGINE = DeltaLake(path, [aws_access_key_id, aws_secret_access_key,])
```

Comment:
it also has optional parameter `format`, but by default delta lake uses parquet, and it will be the default here too.